hw3/01-characters.js

```diff
@@ -0,0 +1,39 @@
+// url for the Thrones API
+const url = "https://thronesapi.com/api/v2/Characters";
+const loader = document.getElementById("loading");
+
+const fetchThrones = async (thronesUrl) => {
+  try {
+    const response = await fetch(Thronesurl);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    console.error("Error:", error);
```

Comment:
console.error is useful for developers and debugging. However, a casual user is likely not going to check the console dev tools.

It would be more user friendly to also display/append an error message on the web page.